wiki/Writing-tests-with-good-behavioral-coverage.md

```diff
@@ -42,75 +42,89 @@ fun getName(name: String? = " ") =
 A basic test case for line coverage might look like this:
 
 ```kotlin
-@Test fun testValidName() {
+@Test 
+fun testValidName() {
   // Tests line coverage by hitting the line where name is accessed
-  assertThat(getName("Alice"), equalTo("Alice")) 
+  assertThat(getName("Alice")).isEqualTo("Alice") 
 }
 ```
 
 **Line Coverage Result:** This test covers the line of code but doesn’t test all scenarios.
 
-### Behavioural Coverage Testing
+### Behavioral Coverage Testing
 
 To ensure behavioral coverage, the test needs to verify various conditions:
 
 ```kotlin
-@Test fun testGetName_withDefaultValue_result(getName()) {
+@Test 
+fun testGetName_withDefaultValue_result(getName()) {
   // Default value when no name is provided
-  assertThat(getName(), equalTo(" ")) 
+  assertThat(getName()).isEqualTo(" ") 
```

Comment:
Nit: can be ``assertThat(getName()).isEmpty()``

Reply:
Done. Updated this and the similar one below.